service/kas/access/rewrap.go

```diff
@@ -2,6 +2,10 @@ package access
 
 import (
 	"bytes"
+
+	"google.golang.org/protobuf/encoding/protojson"
```

Comment:
Regroup these and move this down to the 'non-runtime' package import block below